src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import java.util.Scanner;
 public class Duke {
+    Tasks taskList[] = new Tasks[100];
+    void printWelcome() {
+        System.out.println("Hello I'm Duke\nWhat can I do for you?");
+    }
+    void acceptCommands() {
```

Comment:
Since you're only accepting 1 command, don't have to put plural. acceptCommand is sufficient.